dom/nodes/Node-isConnected.html

```diff
@@ -85,6 +85,22 @@
              [nodes[0], nodes[3]]);
 }, "Test with iframes");
 
+function testInShadowTree(mode) {
+  test(function () {
+    var host = document.createElement("div");
+    document.body.appendChild(host);
+
+    const root = host.attachShadow({ mode });
+    const node = document.createElement("div");
+    root.appendChild(node);
```

Comment:
Maybe add an assert_false before the insertion?

Might be nice to assert_true the root as well.